tests/carrier_account_test.go

```diff
@@ -30,6 +42,36 @@ func (c *ClientTests) TestCarrierAccountCreate() {
 	require.NoError(err)
 }
 
+func (c *ClientTests) TestCarrierAccountCreateWithCustomWorkflow() {
+	assert, require := c.Assert(), c.Require()
+
+	_, err := c.GenerateCarrierAccountWithCustomWorkflow()
+
+	// We're sending bad data to the API, so we expect an error
+	require.Error(err)
+
+	// We expect the error to be an API error
+	assert.Equal(reflect.TypeOf(&easypost.APIError{}), reflect.TypeOf(err))
+
+	// We expect the error to be a 422
+	assert.Equal(422, err.(*easypost.APIError).StatusCode)
+
+	// We expect the error to have sub-errors
+	assert.NotEmpty(err.(*easypost.APIError).Errors)
+
+	// We expect one of the sub-errors to be regarding a missing field
```

Comment:
I'd argue many of these comments are self-explanatory since the code is documenting itself. It may be better to have a single comment at the top of this test describing what's expected as a whole based on the data passing and why instead of commenting each line individually.